h/auth/__init__.py

```diff
@@ -52,19 +80,18 @@ def includeme(config):
     config.include('pyramid_authsanity')
 
     if config.registry.settings.get('h.proxy_auth'):
+        # Set a proxy authentication policy instead of the default policy
         log.warning('Enabling proxy authentication mode: you MUST ensure that '
                     'the X-Forwarded-User request header can ONLY be set by '
                     'trusted downstream reverse proxies! Failure to heed this '
                     'warning will result in ALL DATA stored by this service '
                     'being available to ANYONE!')
+        config.set_authentication_policy(PROXY_POLICY)
 
-        DEFAULT_POLICY = AuthenticationPolicy(api_policy=API_POLICY,
```

Comment:
Seemed like we shouldn't be using constant-style var naming if we're reassigning a value here...I've moved all of the "constants" out of the `includeme` function.